Mage.Sets/src/mage/sets/Jumpstart2022.java

```diff
@@ -126,6 +126,7 @@ private Jumpstart2022() {
         cards.add(new SetCardInfo("Cellar Door", 759, Rarity.UNCOMMON, mage.cards.c.CellarDoor.class));
         cards.add(new SetCardInfo("Cemetery Recruitment", 385, Rarity.COMMON, mage.cards.c.CemeteryRecruitment.class));
         cards.add(new SetCardInfo("Certain Death", 386, Rarity.COMMON, mage.cards.c.CertainDeath.class));
+        cards.add(new SetCardInfo("Chains Of Custody", 3, Rarity.COMMON, mage.cards.c.ChainsOfCustody.class));
```

Comment:
Card name is "Chains **of** Custody", not "Chains **Of** Custody"

Reply:
done